waku/waku_discv5.nim

```diff
@@ -122,6 +122,75 @@ proc new*(T: type WakuDiscoveryV5,
 
   WakuDiscoveryV5.new(rng, conf, some(record))
 
+proc enrSharding(wd: WakuDiscoveryV5, newTopics: seq[PubsubTopic], add: bool):  Result[void, string] =
```

Comment:
Nitpick: perhaps a more verb-y proc name (`modifyEnrShardField` but something better 😛 )

Reply:
Naming is hard...